src/backend/request.rs

```diff
@@ -54,16 +54,18 @@ impl Request {
         );
         server.at(&path, request).await?;
 
-        let response = match fut.await {
-            Err(_) => Response::cancelled(),
-            Ok(response) => Response::ok(response?),
-        };
+        let response = fut.await;
+
         #[cfg(feature = "tracing")]
         tracing::debug!("{_method} returned {:#?}", response);
```

Comment:
This change effectively changes how this call logs. Did you test whether more or less the output is logged?